examples/alloy-autoscaling-and-storage/values.yaml

```diff
@@ -42,4 +42,24 @@ alloy:
           storageClassName: "standard"
           resources:
             requests:
-              storage: 5Gi
\ No newline at end of file
+              storage: 5Gi
+
+alloy-logs:
+  alloy:
+    storagePath: /var/lib/alloy
+    mounts:
+      extra:
+        - mountPath: /var/lib/alloy
+          name: alloy-log-positions
+
+  controller:
+    enableStatefulSetAutoDeletePVC: true
+    volumeClaimTemplates:
+      - metadata:
+          name: alloy-log-positions
+        spec:
+          accessModes: ["ReadWriteOnce"]
+          storageClassName: "standard"
+          resources:
+            requests:
+              storage: 100Mi
```

Comment:
Looks good, just that we haven't been running it this way, so not much experience on our side.